corehq/apps/userreports/es/columns.py

```diff
@@ -0,0 +1,26 @@
+from corehq.apps.reports.datatables import (
+    DataTablesColumn,
+    DataTablesHeader,
+    DTSortType,
+)
+
+
+class EsColumnConfig(object):
```

Comment:
Good point. I don't think it does. I basically ended up copy/pasting that entire file when I was trying to figure out distinct values in ES, so that came with it. Don't think I gave it too much thought since it was in `sql/columns.py`